trade-finance-manager-ui/server/nunjucks-configuration/filter-dashIfEmpty.js

```diff
@@ -5,7 +5,8 @@ const dashIfEmpty = (text) => {
 
   const textStr = String(text);
 
-  const isEmpty = !(textStr && textStr.trim().length > 0 && textStr !== 'NaN' && textStr !== 'Invalid date');
+  const isEmpty = !textStr || textStr.trim().length === 0 || textStr === 'NaN' || textStr === 'Invalid date';
```

Comment:
Can we add a unit test please, doesn't seem to be anything that checks the length specifically